blocks/bricks/__init__.py

```diff
@@ -314,6 +315,75 @@ def _set_dim(self, value):
     input_dim = output_dim = property(_get_dim, _set_dim)
 
 
+class PReLU(Feedforward, Initializable):
```

Comment:
I would say that cutting-edge research bits and pieces belong to `blocks-extras`, not `blocks`.